server/enterprise/message_export/actiance_export/actiance_export.go

```diff
@@ -235,13 +302,100 @@ func channelHasActivity(cmhs []*model.ChannelMemberHistoryResult, startTime int6
 	return false
 }
 
-func postToExportEntry(post *model.MessageExport, createTime *int64, message string) *PostExport {
+func createdPostToExportEntry(post *model.MessageExport) PostExport {
+	userType := "user"
+	if post.IsBot {
+		userType = "bot"
+	}
+	return PostExport{
+		MessageId:    *post.PostId,
+		CreateAt:     *post.PostCreateAt,
+		Message:      *post.PostMessage,
+		UserType:     userType,
+		UserEmail:    *post.UserEmail,
+		PreviewsPost: post.PreviewID(),
+	}
+}
+
+func deletedPostToExportEntry(post *model.MessageExport, newMsg string) PostExport {
+	userType := "user"
+	if post.IsBot {
+		userType = "bot"
+	}
```

Comment:
I am thinking a enum of strings could make sense here, so we get nice named constants.